src/KingSystem/System/Quest.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include "KingSystem/ActorSystem/actActor.h"
+#include "KingSystem/GameData/gdtManager.h"
+#include "KingSystem/Resource/resHandle.h"
+#include "KingSystem/Utils/Byaml/Byaml.h"
+#include "agl/Utils/aglParameter.h"
+
+namespace ksys::qst {
+
+struct QuestStep {
```

Comment:
This was called `Step` according to one string. (There's also ksys::qst::Indicator and ksys::qst::ActorData)